docs/Permissions/Permissioning.md

```diff
@@ -9,23 +9,23 @@ account permissions enabled, or both.
 
 ## Node Whitelisting 
 
-Node whitelisting is specified by the nodes whitelist in the [`permissions_config.toml`](#permissions-configuration-file) file. 
+Node whitelisting is specified by the nodes whitelist in the [permissions configuration file](#permissions-configuration-file) file. 
 A node with node whitelisting enabled communicates only with nodes in the nodes whitelist. 
 
-!!! example "Nodes Whitelist in `permissons_config.toml`"
+!!! example "Nodes Whitelist in Permissions Configuration File"
     `nodes-whitelist=["enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.168.0.9:4567","enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.169.0.9:4568"]`
 
-Node whitelisting is at the node level. That is, each node in the network has a [`permissions_config.toml`](#permissions-configuration-file)
+Node whitelisting is at the node level. That is, each node in the network has a [permissions configuration file](#permissions-configuration-file)
 file in the [data directory](../Reference/Pantheon-CLI-Syntax.md#data-path) for the node.  
 
 To view or update the nodes whitelist when the node is running, use the JSON-RPC API methods:
  
-* [perm_addNodesToWhitelist](../Reference/JSON-RPC-API-Methods.md#perm__addnodestowhitelist)
-* [perm_removeNodesFromWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_removeNodesFromWhiteList)
-* [perm_getNodesWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_getNodesWhiteList)
+* [perm_addNodesToWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_addnodestowhitelist)
+* [perm_removeNodesFromWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_removenodesfromwhitelist)
+* [perm_getNodesWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_getnodeswhitelist)
 
 !!! note
-    Each node has a `permissions_config.toml` file which means nodes can have different nodes whitelists. 
+    Each node has a [permissions configuration file](#permissions-configuration-file) which means nodes can have different nodes whitelists. 
     This means nodes may be participating in the network that are not on the whitelist of other nodes in the network. 
     We recommend each node in the network has the same nodes whitelist. 
```

Comment:
`We recommend each node in the network has the same nodes whitelist.`

This recommendation makes more sense in the accounts whitelist than in the nodes whitelist. I think we can remove this.

Reply:
Leaving as is per @arash009